config/sync/views.view.stanford_events.yml

```diff
@@ -1126,7 +1126,7 @@ display:
             trim: 0
             preserve_tags: ''
             html: 0
-          element_type: h3
+          element_type: h2
```

Comment:
Was there a reason why this was an h3? 🤔 Was it because this view was intended to be used as a block or paragraph type (where an h2 heading exists)? On this page, it goes from h1 to h2.
https://amptesting.sites.stanford.edu/events